src/components/Banner/Banner.module.css

```diff
@@ -73,7 +73,6 @@
   position: absolute;
   top: var(--eds-size-half);
   right: var(--eds-size-half);
-  /* TODO: handle hover styles */
```

Comment:
The hover styles are being handled by the `Button` component now; this was originally using a raw `button` with no hover styling.